frontend/src/components/dashboard/components/categories-list/categories-list.tsx

```diff
@@ -0,0 +1,31 @@
+import { IconName } from 'common/types/types';
+import { ReactElement } from 'react';
+
+import { Category } from './components/category/category';
+import styles from './styles.module.scss';
+
+type Category = {
+  id: number;
+  key: string;
+  name: string;
+};
+
+type Props = {
+  items: Category[];
+};
+
+const CategoriesList = ({ items }: Props): ReactElement => {
+  return (
+    <div className={styles.categoriesList}>
```

Comment:
As we're displaying list of items it would be better to show them as unordered list

Reply:
I do not think, I got that. Seems, that there is no need in Set (my idea about displaying unordered collection of data), then how should it be implemented?

Reply:
I am just about `<ul>` for the container and `<li>` for items, maybe you thought about other thing 🤔

Reply:
Oh, ok, got it!